src/main/java/org/bookstore/dto/BookDto.java

```diff
@@ -0,0 +1,7 @@
+package org.bookstore.dto;
+
+import java.math.BigDecimal;
+
+public record BookDto(Long id, String title, String author, String isbn,
+                       BigDecimal price, String description, String coverImage) {
```

Comment:
minor 

usually records are formatted as follows:
```suggestion
public record BookDto(Long id, 
                      String title, 
                      String author, 
                      String isbn,
                      BigDecimal price, 
                      String description, 
                      String coverImage) {
```